app/src/main/res/values-land/dimens.xml

```diff
@@ -307,4 +307,9 @@
   <dimen name="promoted_story_card_layout_margin_start">8dp</dimen>
   <dimen name="promoted_story_card_text_size">14sp</dimen>
   <dimen name="promoted_story_card_margin_top">8dp</dimen>
+
+  <!-- Topic Lessons Title -->
+  <dimen name="topic_lessons_title_marginStart">100dp</dimen>
```

Comment:
Rename this and all other dimens name to `topic_lessons_title_margin_start `.
Basically all names should contain only small-characters separated by `_`.

Reply:
@ayaazansari PTAL